dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -445,17 +445,17 @@ Sedp::init(const RepoId& guid,
 #endif
 
   if (spdp_.available_builtin_endpoints() & DISC_BUILTIN_ENDPOINT_PARTICIPANT_ANNOUNCER) {
-    participant_message_writer_->enable_transport_using_config(reliable, durable, transport_cfg_);
```

Comment:
Doesn't this violate 8.4.13.3 of the RTPS spec?

Reply:
I'll defer to @mitza-oci

Reply:
Currently the implementation of this built-in endpoint in SEDP is not Durable, but we are enabling Durability in the transport.

The part of the spec you reference does say it should be Durable.  A peer process wouldn't be able to tell that we're not enabling Durability here, it would just appear that there are no data samples in the cache for a late-joining reader.

We could also consider implementing Durability, but it seems to have limited value for the BuiltinParticipantMessageWriter.

Reply:
In particular we found that nothing adds elements to the `local_participant_messages_` collection.

However, after more discussions I think the current plan is to keep it durable.